src/ext/ddtrace.c

```diff
@@ -1523,6 +1615,7 @@ static const zend_function_entry ddtrace_functions[] = {
     DDTRACE_SUB_NS_FE("Config\\", integration_analytics_enabled, arginfo_ddtrace_config_integration_analytics_enabled),
     DDTRACE_SUB_NS_FE("Config\\", integration_analytics_sample_rate,
                       arginfo_ddtrace_config_integration_analytics_sample_rate),
+    DDTRACE_SUB_NS_FE("Testing\\", trigger_error, arginfo_ddtrace_testing_trigger_error),
```

Comment:
I think a runtime variable makes more sense if we do anything here. We want to run tests using the official packages, which wouldn't be built with such compile-time flags.

Currently `_DD_LOAD_TEST_INTEGRATIONS` is in use by `_dd_load_test_integrations` (which took me a bit to find because I reworked all of that in the [pending deferred loading PR](https://github.com/DataDog/dd-trace-php/pull/1006)).

Any other thoughts based on this info?

Reply:
Good point about official builds. I can't think of another alternative that makes sense so we can just keep it as is. :)